### PR TITLE
dstart and until should be same type (floating or localized)

### LINKED ISSUE #810: khal new --repeat --until produces invalid VEVENTs
DTSTART and UNTIL are sometimes not of the same type (floating / localized)

### PR DESCRIPTION
**Pull request by allanwind**
Fix #810

### CODE REVIEW

**Comment by allanwind** [comment]
Ticket was written as a cli issue.  That said, I also looked very closely at the class LocalizedEvent which localizes start and end as needed which comes into play via Event.fromVEvents().  Both import_event and pritnics call Event.fromVEvents().

File: tests/parse_datetime_test.py [hunk 1]
```diff
@@ -497,7 +497,7 @@ def test_description():
          'DTSTART;TZID=Europe/Berlin;VALUE=DATE-TIME:20140216T080000',
          'DTEND;TZID=Europe/Berlin;VALUE=DATE-TIME:20140216T090000',
          'DESCRIPTION:please describe the event',
-         'RRULE:FREQ=DAILY;UNTIL=20150605T000000')))
+         'RRULE:FREQ=DAILY;UNTIL=20150604T230000Z')))
```
Comment (WhyNotHugo) [inline tests/parse_datetime_test.py:500]: Do you think it makes sense to create a copy of this test with non-localised times?

**Review by WhyNotHugo — APPROVED** [review]
Makes sense to me.

File: tests/parse_datetime_test.py (hunk 1 above)
Comment (allanwind) [reply tests/parse_datetime_test.py:523]: Done.

**Comment by WhyNotHugo** [comment]
Thanks!

**Pull request merged**